synapse/storage/schema/main/delta/80/02_scheduled_tasks.sql

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C
```

Comment:
Why do we have two files adding the same table?

Reply:
Failure at merging properly ^^